config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,87 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file eases your Rails 7.1 framework defaults upgrade.
+#
+# Uncomment each configuration one by one to switch to the new default.
+# Once your application is ready to run with all new defaults, you can remove
+# this file and set the `config.load_defaults` to `7.1`.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+# https://guides.rubyonrails.org/upgrading_ruby_on_rails.html
+
+# 💥✅ I've removed the lines in this file where I agree with the new defaults
+# or when they are not applicable to us. I've kept the lines that I'm not sure about
+# or that I think we should discuss.
+# TODO: We still have to activate the new defaults by setting
+# `config.load_defaults` to `7.1` (!)
+
+# ❓ Parameters Hash -- Not sure whether this affects us
+###
+# Do not treat an `ActionController::Parameters` instance
+# as equal to an equivalent `Hash` by default.
+#++
+# Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
+
+# ❓ Invalid cache expiration time, what behavior do we want?
+###
+# Specify if an `ArgumentError` should be raised if `Rails.cache` `fetch` or
+# `write` are given an invalid `expires_at` or `expires_in` time.
+# Options are `true`, and `false`. If `false`, the exception will be reported
+# as `handled` and logged instead.
+#++
+# Rails.application.config.active_support.raise_on_invalid_cache_expiration_time = true
```

Comment:
I think setting this to the new default `true` will be fine. What do you think?

Reply:
I agree.